packages/kbn-apm-synthtrace/src/scenarios/other_bucket_group.ts

```diff
@@ -26,11 +28,22 @@ const scenario: Scenario<ApmFields> = async ({ logger, scenarioOpts }) => {
 
       const serviceRange = [
         ...lodashRange(0, numServices).map((groupId) => `service-${groupId}`),
-        '_other',
+        OVERFLOW_BUCKET_NAME,
       ];
 
+      const overflowCountConfig: Record<string, number> = {
+        [OVERFLOW_BUCKET_NAME]: 20,
+      };
+
       const instances = serviceRange.flatMap((serviceName) => {
-        const services = ENVIRONMENTS.map((env) => apm.service(serviceName, env, 'go'));
+        const services = ENVIRONMENTS.map((env) =>
+          apm.service({
+            name: serviceName,
+            environment: env,
+            agentName: 'go',
+            'service_transaction.aggregation.overflow_count': overflowCountConfig[serviceName] ?? 0,
```

Comment:
This should only be on service transaction metrics no? This should be implemented in the aggregators I think.

Reply:
I have this field added in the service metric aggregator here - https://github.com/elastic/kibana/pull/151908/files#diff-78a403b68ebbccacdeb82a1cfc1a7eb431e88af7110a21c48026530f4a062c6a

Now we need to be able pass the value from the scenario file. So when I add it to the service here, i see the aggregator is able to pick up the value.

Reply:
What i understood is, we need to be able to pass this value somewhere in the `event`.
So when the `createApmMetricAggregator` runs, it will pick the specified keys [here](https://github.com/elastic/kibana/blob/7d162009f03db3948b4476941370f567f4b9c189/packages/kbn-apm-synthtrace/src/lib/apm/aggregators/create_service_metrics_aggregator.ts#L13-L20) from the list of keys in the event and set the metrics

Reply:
Another alternative is to check in the aggregators for `_other` in the event name and add a hardcoded value

For [transactions](https://github.com/elastic/kibana/blob/a153acc7f030e7f0303e8d1e4d3f620befc015c0/packages/kbn-apm-synthtrace/src/lib/apm/aggregators/create_transaction_metrics_aggregator.ts#L83)
```
'transaction.aggregation.overflow_count: event['transaction.name'] === `_other` ? hardcoded_value : 0
```

For [services](https://github.com/elastic/kibana/blob/a153acc7f030e7f0303e8d1e4d3f620befc015c0/packages/kbn-apm-synthtrace/src/lib/apm/aggregators/create_service_metrics_aggregator.ts#L49):
```
'service_transaction.aggregation.overflow_count: event['service.name'] === `_other` ? hardcoded_value : 0
```

Reply:
@yngrdyn The alternative is good, but i then see other problems -
1. We will have a fixed `hardcoded` value always irrespective of scenario. I was hoping if we can control this value from scenario itself.
2. Also if this list increases, we will keep on adding items at the end here. Instead let's maintain the list [here](https://github.com/elastic/kibana/blob/7d162009f03db3948b4476941370f567f4b9c189/packages/kbn-apm-synthtrace/src/lib/apm/aggregators/create_service_metrics_aggregator.ts#L13-L20) and then let scenario pass whatever they want ?

Reply:
For anyone coming to this comment in future, we had a zoom discussion and Dario provided a very good suggestion to enhance the Synthtrace client in such way that it can replicate APM Server behaviour. This way we don't have to pass such metrics values from scenario itself, but from a configuration values. Synthtrace based on the provided configuration will generate those metrics while processing events.